src/azul/deployment.py

```diff
@@ -296,6 +307,27 @@ def resource(self, *args, **kwargs):
 aws = AWS()
 
 
+class Terraform:
+
+    @cached_property
+    def taggable_resources(self) -> Sequence[str]:
+        with open(config.tracked_schema_path) as f:
+            schema = json.load(f)['schema']
```

Comment:
Don't like how the knowledge about the internal format of that file is distributed over the codebase. Consolidate the reading and writing of the file in a class, probably this one.